yt/visualization/tests/test_callbacks_geographic.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from yt import SlicePlot
+from yt.testing import assert_fname, fake_amr_ds, requires_module
+
+
+@requires_module("cartopy")
+@pytest.mark.parametrize("geometry", ["geographic", "internal_geographic"])
+def test_quiver_callback_geographic(geometry, tmp_path):
```

Comment:
we could make this an actual image test. The documentation for how to do that is still not merged (I think I need to finish the PR first ...) but it's there: https://github.com/yt-project/yt/pull/4418

Reply:
yeah, it'd be a good image test. I'll switch it over after I revise the test data a bit.